bbb-learning-dashboard/src/components/PluginsTable.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import { DataGrid } from '@mui/x-data-grid';
+import UserAvatar from './UserAvatar';
+
+// Type of a plugins.genericDataForLearningAnalyticsDashboard is of the form: {
+//  columnTitle: string;
+//  value: string
+// }
+const PluginsTable = (props) => {
+  const {
+    pluginsColumnTitle,
+    allUsers, intl, pluginCardTitle,
+  } = props;
+
+  const commonUserProps = {
+    field: 'User',
+    headerName: intl.formatMessage({ id: 'app.learningDashboard.pluginsTable.userLabel', defaultMessage: 'User' }),
+    flex: 1,
+    sortable: true,
+  };
+
+  const commonCountProps = {
+    field: 'count',
+    headerName: intl.formatMessage({ id: 'app.learningDashboard.pluginsTable.answerTotal', defaultMessage: 'Total' }),
+    flex: 1,
+    sortable: true,
+  };
```

Comment:
Do `app.learningDashboard.pluginsTable.userLabel` and `app.learningDashboard.pluginsTable.answerTotal` exist?

And I think it should not show "Total" once it accepts any string as value. If you want to show Total the plugin will need to send a column with title "Total" and its value.

Reply:
Actually, that is the total number of generic-data for that specific User. But I agree on changing the id of the locale.